src/irmin-pack/unix/mapping_file.ml

```diff
@@ -436,6 +439,8 @@ module Make (Errs : Io_errors.S with module Io = Io.Unix) = struct
       else
         let off, len =
           (* Decoding a pair of int can't fail *)
+          (* Bytes.unsafe_to_string usage: possibly safe TODO justify safety, or convert
```

Comment:
I _think_ this is safe because the string is immediately decoded with `decode_bin_pair` and then never used again. Another way to say it: the buffer cannot be modified while the string is being used for decoding.